### PR TITLE
Add content warning to modlog and fix modlog routing bug

### PR DESCRIPTION
**Pull request by SleeplessOne1917**
This is the PR for #991. 

This is what it looks like on a desktop:
modlog-pc

This is what it looks like on mobile:
modlog-mobile

Translation PR will be made soon.

In addition to the content warning, I also fixed a bug I introduced in the query param PR regarding routing for community specific modlogs.

### CODE REVIEW

File: src/assets/css/main.css [hunk 1]
```diff
@@ -140,8 +140,8 @@
 }
 
 .icon-emoji-admin {
-  max-width: 24px; 
-  max-height: 24px; 
+  max-width: 24px;
+  max-height: 24px;
```
Comment (dessalines) [inline src/assets/css/main.css:144]: Thanks. We should alter prettier / to make sure it works for css files too, I'll add that now.

File: src/shared/components/community/sidebar.tsx [hunk 2]
```diff
@@ -251,7 +251,7 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
         <li className="list-inline-item">
           <Link
             className="badge badge-primary"
-            to={`/modlog/community/${this.props.community_view.community.id}`}
+            to={`/modlog/${this.props.community_view.community.id}`}
```
Comment (dessalines) [inline src/shared/components/community/sidebar.tsx:254]: Oops, my bad. Thanks for catching.

File: src/shared/components/modlog.tsx [hunk 3]
```diff
@@ -782,6 +782,18 @@ export class Modlog extends Component<
...
+          <div
+            className="alert alert-warning text-sm-start text-xs-center"
+            role="alert"
+          >
+            <Icon
+              icon="alert-triangle"
+              inline
+              classes="mr-sm-2 mx-auto d-sm-inline d-block"
+            />
+            <strong>CONTENT WARNING</strong>: Some deleted posts may contain
```
Comment (dessalines) [inline src/shared/components/modlog.tsx:794]: Language looks good to me.

**Review by dessalines — COMMENTED** [review]
Once you get the translation added, update this and I can merge.

File: src/shared/components/community/sidebar.tsx (hunk 2 above)
Comment (SleeplessOne1917) [reply src/shared/components/community/sidebar.tsx:254]: TBH I think I might have introduced that bug. That route was correct until the query param change I made.

**Comment by dessalines** [comment]
K that's merged.

**Review by dessalines — APPROVED** [review]

**Pull request merged**